apps/src/applab/applab.js

```diff
@@ -139,6 +141,10 @@ function stepDelayFromStepSpeed(stepSpeed) {
 }
 
 function loadLevel() {
+  // Only send PROJECT_ACTIVITY event for students or non-authenticated users
+  if (getStore().getState().currentUser?.userType !== 'teacher') {
+    analyticsReporter.sendEvent(EVENTS.PROJECT_ACTIVITY, {}, PLATFORMS.BOTH);
```

Comment:
If we find value in it, we could include the project type for most of these in the payload, something like `project_type: app_lab`. P5Lab could be tricky as it includes a few project types.

Reply:
Note that RED hasn't asked for this, so it may be something we instrument later if the ask comes in.